tests/integration/basic_c.cpp

```diff
@@ -164,4 +166,6 @@ TEST_F(MemgraphConnection, InsertAndRetriveFromMemegraph) {
   ASSERT_EQ(rows, 1);
   ASSERT_EQ(status, 0);
   ASSERT_EQ(mg_session_commit_transaction(session, &result), 0);
+
+  std::cout << "END END" << std::endl;
```

Comment:
?

Reply:
Nice catch!

Reply:
> Maybe have in src windows/ and posix/ folders for platform specific source files so we don't complicate the names and clutter the src folder?

I was thinking about that, but `mg_socket` should be the only platform-dependent code + there should be 2, maybe 3 implementations (at least in the foreseeable future). I would leave it as is for now and then reevaluate when the support for, e.g., Mac comes in.